analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -23,6 +23,7 @@ import ch.frankel.slf4k.*
 
 import com.here.ort.analyzer.managers.Unmanaged
 import com.here.ort.downloader.VersionControlSystem
+import com.here.ort.downloader.vcs.GitRepo
 import com.here.ort.model.AnalyzerResultBuilder
```

Comment:
Pull request: it seems the paths to the definition files one can specify in the .ort.yml e.g. for project exludes are relative to the root of the git repository. This means the paths become ambigious e.g. when you have multiple git repositories with a build.gradle in the top level directory. On option could be to make the paths relative to the repo root dir, or alternatively to add an attribute to the project exclude like:
projects
   path: "path inside git repo"
   git-repository-url: "gerrit.it.here.com/mos/main/mw

I fear that without addressing this, this new feature is not really usable. Whaqt do you think?

Reply:
The root problem is that the result file does not contain the information where inside a Git-Repo project the project is located, so we could not match `path inside git repo` against anything.
However, the change is still useful for all cases except for project excludes for two project with the same `definitionFilePath`.
As discussed offline, I will merge this and we will address this issue later.